app/src/main/java/invoice_automation/utils/QuickBooksUtils.java

```diff
@@ -0,0 +1,60 @@
+package invoice_automation.utils;
+
+import com.intuit.ipp.data.Customer;
+import com.intuit.ipp.data.EmailAddress;
+import com.intuit.ipp.data.PhysicalAddress;
+import com.intuit.ipp.data.TelephoneNumber;
+import invoice_automation.model.Address;
+import invoice_automation.model.School;
+
+public class QuickBooksUtils {
+    /**
+     * Basic utility functions to reduce levels of complexity when
+     * working with QuickBooks
+     * */
+
+
+    /**
+     * Derives a new customer object from a school object
+     * @param school the school object whose values are used in the creation of a new customer
+     * @return a new customer created from the school
+     * */
+    public static Customer getCustomerFromSchool(School school) {
+        Customer newCustomer = new Customer();
+        newCustomer.setCompanyName(school.getSchoolName());
+        newCustomer.setDisplayName(school.getSchoolName());
+        if (!school.getEmail().equals(null)) {
+            newCustomer.setPrimaryEmailAddr(new EmailAddress());
+            newCustomer.getPrimaryEmailAddr().setAddress(school.getEmail());
+        } else { newCustomer.setPrimaryEmailAddr(null); }
+        if (!school.getPhoneNumbers().equals(null)) {
```

Comment:
phone numbers is also @NonNull, so do an isEmpty() check here instead

Reply:
nit: put the result of getPhoneNumbers() in a variable so we don't keep calling it